horovod/run/horovod_task_fn.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Uber Technologies, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import sys
+
+from horovod.run.task import horovod_task_service
+from horovod.run.driver import horovod_driver_service
+from horovod.run.common.util import codec, host_hash
+
+
+def _task_fn(index, driver_addresses, num_hosts, tmout, key):
+    hvdtask_service = horovod_task_service.HorovodRunTaskService(index, key)
```

Comment:
Could use similar naming to Spark code, just call it "task"

Reply:
done